### DISCUSSION TITLE
V5 Documentation

### DISCUSSION THREAD

**Discussion by rafaphp** [phalcon/cphalcon#15949] [General]
The V5 documentation is not complete on the website, is there any prediction of when it will be updated?

**Comment by niden** [comment]
We have one major thing to address (PSR-7) and after that it will be all documentation work.

I cannot reliably say when that will start since there are a lot of things going on with real life at the moment but I will try my best to get this out as soon as I can.

**Reply by rafaphp** [reply]
Thanks for answering!
